lib/metadata_json_deps/runner.rb

```diff
@@ -17,6 +17,17 @@ def initialize(filename, updated_module, updated_module_version, verbose, use_sl
     end
 
     def run
+      begin
+        @forge.get_mod(@updated_module.sub('/', '-'))
+        SemanticPuppet::Version.parse(@updated_module_version)
+      rescue StandardError => error
+        message = "Updated module name *#{@updated_module}* or updated version *#{@updated_module_version}* is incorrect. Please check README file! \n"
```

Comment:
I would change the wording of the message to be a bit more explicit to the user. Something like "Error: Verify *#{@updated_module}* exists on Puppet Forge and correct semantic versioning syntax is used."